tasks/move-granules/tests/test-index.js

```diff
@@ -665,6 +665,78 @@ test.serial('url_path is evaluated correctly when the metadata file is ISO', asy
   t.true(check);
 });
 
+test.serial('url_path is evaluated correctly when collection is configured to use .cmr.xml as metadata file', async (t) => {
```

Comment:
I think we may need unit coverage of the `updateGranuleMetadata` call that's downstream of the changed logic in `index.js` - we're not validating the correct updates are made to the cmr file, or that the call is at least made.  That seems like a functional output of running the lambda we should check.    

Also the explicit case that's in the ticket (cmr.json/iso.xml) isn't being directly covered, although that's perhaps more of a config nit.

Reply:
The unit test verifies that the files are moved to the correct destination